src/Modules/Admin/Commands.Admin/help/Add-PowerBIEncryptionKey.md

```diff
@@ -0,0 +1,116 @@
+---
+external help file: Microsoft.PowerBI.Commands.Admin.dll-Help.xml
+Module Name: MicrosoftPowerBIMgmt.Admin
+online version:
+schema: 2.0.0
+---
+
+# Add-PowerBIEncryptionKey
+
+## SYNOPSIS
+Adds an encryption key
+
+## SYNTAX
+
+### Default (Default)
+```
+Add-PowerBIEncryptionKey -Name <String> -KeyVaultKeyUri <String> [-Default] [<CommonParameters>]
+```
+
+### DefaultAndActivate
+```
+Add-PowerBIEncryptionKey -Name <String> -KeyVaultKeyUri <String> [-Default] [-Activate] [<CommonParameters>]
+```
+
+### Activate
+```
+Add-PowerBIEncryptionKey -Name <String> -KeyVaultKeyUri <String> [-Activate] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The commandlet will add an encryption key.
+
+## EXAMPLES
+
+### Example 1
+```
+PS C:\> Add-PowerBIEncryptionKey -Name 'testName' -KeyVaultKeyUri 'Uri' -Default -Activate
+```
+
+## PARAMETERS
+
+### -Activate
+If the Activate flag is $true, all of the capacities that don't have activated the BYOK functionality will immediately use this key
```

Comment:
For switches, look at doc for other cmdlets that have switch such as -AsString https://docs.microsoft.com/en-us/powershell/module/microsoftpowerbimgmt.profile/get-powerbiaccesstoken?view=powerbi-ps